### PR TITLE
update contact information on the repo and release package

### PR DESCRIPTION
**Pull request by ekzhu**
(1) Replacing an individual outlook.com email address with a mailing list containing all current repo maintainers
(2) Remove text and keep Microsoft open-source statement

### CODE REVIEW

**Review by gagb — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -97,7 +97,7 @@
     name="pyautogen",
     version=__version__,
     author="AutoGen",
-    author_email="auto-gen@outlook.com",
+    author_email="autogen-contact@service.microsoft.com",
```
Comment (qingyun-wu) [inline setup.py:100]: Would like to know why the changes are needed.

**Review by qingyun-wu — DISMISSED** [review]

File: setup.py (hunk 1 above)
Comment (gagb) [reply setup.py:100]: Hi Qingyun, none of the other maintainers have access to this private outlook.com address. From a security and transparency standpoint microsoft/autogen needs a better distribution list. You are are already on this new address.

**Review by qingyun-wu — DISMISSED** [review]
Thanks Gagan and Eric! I think we would need more reviewers on this since these changes are critical. Do you mind waiting a bit? I need to at least ensure all maintainers are on this new address.

**Review by jackgerrits — APPROVED** [review]

**Review by LittleLittleCloud — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (thinkall) [inline setup.py:100]: Thank you @ekzhu , @gagb and @qingyun-wu .  Would you like to add all the non-MS maintainers to the distribution list?

File: README.md [hunk 2]
```diff
@@ -68,11 +68,6 @@
 
 AutoGen is an open-source programming framework for building AI agents and facilitating cooperation among multiple agents to solve tasks. AutoGen aims to streamline the development and research of agentic AI, much like PyTorch does for Deep Learning. It offers features such as agents capable of interacting with each other, facilitates the use of various large language models (LLMs) and tool use support, autonomous and human-in-the-loop workflows, and multi-agent conversation patterns.
 
-**Open Source Statement**: The project welcomes contributions from developers and organizations worldwide. Our goal is to foster a collaborative and inclusive community where diverse perspectives and expertise can drive innovation and enhance the project's capabilities. Whether you are an individual contributor or represent an organization, we invite you to join us in shaping the future of this project. Together, we can build something truly remarkable.
```
Comment (yiranwu0) [inline README.md:71]: I don't think this needs to be deleted. We are still open-sourcing right?

**Review by yiranwu0 — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (ekzhu) [reply README.md:71]: Microsoft open-source statement: https://github.com/microsoft/autogen/tree/main?tab=readme-ov-file#contributing

Comment (gagb) [reply README.md:71]: @yiranwu0 -- this text is redundant with contributing/oss policy text at the bottom of the readme, which hasn't changed!

File: samples/tools/autogenbench/pyproject.toml [hunk 3]
```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "autogenbench"
 authors = [
-  { name="Autogen Team", email="auto-gen@outlook.com" },
```
Comment (randombet) [inline samples/tools/autogenbench/pyproject.toml:8]: Why change this? Does auto-gen@outlook.com not work anymore?

File: setup.py (hunk 1 above)
Comment (gagb) [reply setup.py:100]: @thinkall, @qingyun-wu who is a non-MS maintainer is already on the list.

File: samples/tools/autogenbench/pyproject.toml [hunk 4]
```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
...
+  { name="Autogen Team", email="autogen-contact@service.microsoft.com" },
```
Comment (randombet) [inline samples/tools/autogenbench/pyproject.toml:8]: This is new. Who owns this?

Comment (gagb) [reply samples/tools/autogenbench/pyproject.toml:8]: @randombet -- this new address for microsoft/autogen is owned by Microsoft.

File: samples/tools/autogenbench/pyproject.toml (hunk 3 above)
Comment (gagb) [reply samples/tools/autogenbench/pyproject.toml:8]: Please see other comments up on the thread.

File: README.md [hunk 5]
```diff
@@ -68,11 +68,6 @@
...
-
-The project is currently maintained by a [dynamic group of volunteers](https://butternut-swordtail-8a5.notion.site/410675be605442d3ada9a42eb4dfef30?v=fa5d0a79fd3d4c0f9c112951b2831cbb&pvs=4) from several different organizations. Contact project administrators Chi Wang and Qingyun Wu via auto-gen@outlook.com if you are interested in becoming a maintainer.
```
Comment (randombet) [inline README.md:73]: I don't like this. Moving key contributors from the top level readme to some corner buried in the docs is discouraging.

**Review by randombet — CHANGES_REQUESTED** [review]
This change need more context and supporting evidence to justify itself.
It contains changes that may jeopardize the contributor community.

File: setup.py (hunk 1 above)
Comment (thinkall) [reply setup.py:100]: Thank you @gagb , are there any other non-MS maintainers or collaborators you think  worth adding to the list?

File: README.md (hunk 5 above)
Comment (ekzhu) [reply README.md:73]: Thanks for the suggestion. We will make sure to acknowledge the key contributors but not using a Notion site through an external link.

Comment (gagb) [reply README.md:73]: Yes contributors will definitely be acknowledged in a follow up PR.

**Comment by gagb** [comment]
> This change need more context and supporting evidence to justify itself. It contains changes that may jeopardize the contributor community.

@randombet  -- a PR for acknowledging contributors in the repo is already on the way. Stay tuned.

File: setup.py (hunk 1 above)
Comment (gagb) [reply setup.py:100]: That's a good question. Can you send suggestions of names? Also the see PR https://github.com/microsoft/autogen/pull/3384 that adds CONTRIBUTORS.md instead of external Notion-based link.
